xrpl/transaction/metadata_builder.go

```diff
@@ -0,0 +1,94 @@
+package transaction
+
+import "github.com/Peersyst/xrpl-go/xrpl/transaction/types"
+
+// TxMetadataBuilder contains all `meta` transaction response fields and
+// enables specific transaction metadata building.
+type TxMetadataBuilder struct {
+	AffectedNodes []AffectedNode `json:"AffectedNodes"`
```

Comment:
Can we extend from `TxObjMeta` ?

Reply:
No, because it won't parse the metadata properly